diff --git a/node_modules/@turnkey/sdk-react/dist/contexts/TurnkeyContext.mjs b/node_modules/@turnkey/sdk-react/dist/contexts/TurnkeyContext.mjs
index 5f8f56a..d947301 100644
--- a/node_modules/@turnkey/sdk-react/dist/contexts/TurnkeyContext.mjs
+++ b/node_modules/@turnkey/sdk-react/dist/contexts/TurnkeyContext.mjs
@@ -125,9 +125,9 @@ var TurnkeyProvider = function TurnkeyProvider(_ref) {
         break;
       case AuthClient.IndexedDb:
         expiry = (session === null || session === void 0 ? void 0 : session.expiry) || 0;
-        if (expiry > Date.now() && session !== null && session !== void 0 && session.token) {
+        // if (expiry > Date.now() && session !== null && session !== void 0 && session.token) {
           setClient(indexedDbClient);
-        }
+        // }
         break;
     }
   }, [session, authIframeClient, passkeyClient, walletClient, indexedDbClient]);
