diff --git a/node_modules/@turnkey/sdk-browser/dist/utils.mjs b/node_modules/@turnkey/sdk-browser/dist/utils.mjs
index ad208ab..9505994 100644
--- a/node_modules/@turnkey/sdk-browser/dist/utils.mjs
+++ b/node_modules/@turnkey/sdk-browser/dist/utils.mjs
@@ -35,7 +35,8 @@ function parseSession(token) {
         sessionType,
         userId,
         organizationId,
-        expiry: exp,
+        // normalize to milliseconds
+        expiry: exp < 10000000000 ? exp * 1000 : exp,
         token: publicKey,
     };
 }
